data/templates/default.asciidoctor

```diff
@@ -0,0 +1,38 @@
+$if(titleblock)$
+= $title$
+$if(author)$
+$for(author)$$author$$sep$; $endfor$
+$endif$
+$if(date)$
+$date$
+$endif$
+$if(keywords)$
+:keywords: $for(keywords)$$keywords$$sep$, $endfor$
+$endif$
+$if(lang)$
+:lang: $lang$
+$endif$
+$if(toc)$
+:toc:
+$endif$
+
```

Comment:
asciidoctor manual says that when latex math is used,
```
:stem: latexmath
```
is needed in the title block.  We could have a variable in WriterState that keeps track of whether math has been used and sets a variable in the template that causes this to be inserted.

I don't know if something similar is also needed for asciidoc.

Reply:
done